src/app/showcase/avatar-showcase/avatar-showcase.component.ts

```diff
@@ -8,7 +10,40 @@ declare var require: any;
 })
 export class AvatarShowcaseComponent implements OnInit {
   exampleHtml: string = require('../../examples/avatar-example/avatar-example.component.html');
-
+  properties: ShowcaseProperty[] = [
+    {
+      name: 'imageSrc',
+      description: 'Points to the src of the image location',
+      defaultValue: 'null',
+      inputValues: ['string'],
+    },
+    {
+      name: 'size',
+      description: 'Sets the size of the button.',
+      defaultValue: 'null',
+      inputValues: ['small', 'medium', 'large'],
```

Comment:
Use sizes from size directive to avoid hardcoded sizes: 
import { Sizes } from '@kirbydesign/designsystem/directives/size/size.directive';
Object.values(this.sizes)